modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJavaSetResultsAndTotalCheck.testjava

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.account.admin.web.internal.dao.search;
+
+import java.util.Objects;
+
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradeJavaSetResultsAndTotalCheck {
+
+	public static SearchContainer<AccountGroupDisplay> create(
+			LiferayPortletRequest liferayPortletRequest,
+			LiferayPortletResponse liferayPortletResponse)
+		throws PortalException {
+
+		SearchContainer searchContainer = new SearchContainer();
+
+		searchContainer.setTotal(new Long(
+		ConsumerLocalServiceUtil.countConsumerByCriterias(
+		this.keywords, this.status)).intValue());
```

Comment:
@TamyrisTorres, please format according Liferay SF pattern. I mean, use tab, blank line accordingly, etc

Tip: run `gw formatSource` on this code